generate-osm/db-utils/find_coordinates.go

```diff
@@ -0,0 +1,317 @@
+package DBUtils
+
+import(
+	"math"
+	"errors"
+	"strconv"
+	"sort"
+	OSMUtil "transform-osm/osm-utils"
+)
+
+type nodePair struct {
+	node1 *OSMUtil.Node
+	node2 *OSMUtil.Node
+	dist float64
+	remDist1 float64
+	remDist2 float64
+}
+
+const r = 6371.0
+var osmData *OSMUtil.Osm

```

Comment:
I am not sure if its the best idea to safe the osmData globally. Why not as a parameter?